### ISSUE TITLE
Add DN training into the original DETR

### ISSUE THREAD

**Issue by chenhbo** [IDEA-Research/DN-DETR#60]
Hi, there. Thank you for your NICE work!

Can we directly add the DN training strategy to the original DETR? 

In the original DETR, there has no reference point to use.
Could u  pls give some suggestions?

**Comment by FengLi-ust** [comment]
Yes, we have tried in our extended DN-DETR version. Please refer to our [new arxiv paper](https://arxiv.org/pdf/2203.01305.pdf)

**Comment by anaelley** [comment]
thanks! Is the code for DN-DETR available?
